net/eoip/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2006-2012 OpenWrt.org
```

Comment:
Copyright years look strange. Is this 8 years old? 
Is the package being imported from the oldpackages repo? If so, that should be mentioned.

Is the upstream still alive?
Is the actual underlying package still being developed?  (or is this old abandonware?)

Reply:
no, it's just a fork that I was working on https://github.com/bogdik/openwrt-linux-eoip